s3/costs/prepare_response.go

```diff
@@ -108,7 +108,7 @@ func parseBuckets(buckets BucketsInfo, parsedDocument bucket, resultType string)
 		if isValidBucket(bucketName) {
 			bucketInfo := getBucketInfoByName(buckets, bucketName)
```

Comment:
`bucketInfo` could be created directly after the condition below, couldn't it?

Reply:
Done.